src/Nuget.PackageIndex.VisualStudio/Analyzers/ProjectTargetFrameworkProviderExporter.cs

```diff
@@ -175,6 +176,16 @@ private void OnAfterSolutionClosing()
             ClearCache();
         }
 
+        private void OnProjectRemoved(Project project)
+        {
+            if (project == null)
```

Comment:
well we receive it from the event - where it said that it must be non null? :) Just to be safe ...